jsignpdf/src/main/resources/net/sf/jsignpdf/translations/messages.properties

```diff
@@ -145,102 +145,102 @@ gui.vs.acro6layers.checkbox=Acrobat 6 layer mode
-hlp.footer.exitCodes=Program exit codes:\r\n-------------------\r\n  0 program finished without error\r\n  1 command line is in a wrong format\r\n  2 no operation requested - e.g. no file for signing provided\r\n  3 signing of some - but not all - files failed\r\n  4 signing of all files failed
-hlp.hashAlgorithm=hash algorithm used for signature. Default value is SHA1. Available values are {0}
-hlp.header=JSignpdf is an application designed to digitally sign PDF documents. If you start the program without any command line argument, the GUI will be started, otherwise you can use JSignPdf in command line batch mode.
-hlp.help=prints this help screen
-hlp.imgPath=image path for visible signature
-hlp.keyAlias=name (alias) of the key, which should be used for signing the document. If this option is not given, the first key in the keystore is used. (List the key aliases using -lk)
-hlp.keyIndex=zero based index of the key, which should be used for signing the document. If neither this option nor alias is given, the first key (index=0) in the keystore is used. (List the key aliases using -lk). This option has lower priority than alias.
-hlp.keyPwd=password of the key in keystore. In most cases you don't need to set this option - only keystore is protected by a password, but just in case :)
-hlp.ksFile=sets KeyStore file - as the value use the path on which is file with private key(s) located (.p12, .pfx, .jks, ...). Some keystores haven't keys stored in a file (e.g. windows keystore - WINDOWS-MY), then don't use this option.
-hlp.ksPwd=password to KeyStore
-hlp.ksType=sets KeyStore type (you can list possible values for this option -lkt argument)
-hlp.l2Text=signature text, you can also use placeholders for signature properties (${signer}, ${certificate}, ${timestamp}, ${location}, ${reason}, ${contact})
-hlp.l2TextFontSize=font size for visible signature text, default value is {0}
-hlp.l4Text=status text
-hlp.listKeys=lists keys in choosen keystore
-hlp.listKsTypes=lists keystore types, which can be used as values -kst option
-hlp.loadProperties=Loads properties from a default file (created by GUI application).
-hlp.loadPropertiesFile=Loads properties from the given file. The file can be create by copying the default property file .JSignPdf created by the GUI in the user home directory.
-hlp.location=location of a signatue (e.g. Washington DC). Empty by default.
+hlp.encryption=Encryption mode for the output PDF default value is NONE. Possible values are {0}. Use togethter with -upwd and -opwd in case of PASSWORD mode, and -ec in case of CERTIFICATE
```

Comment:
```suggestion
hlp.encryption=Encryption mode for the output PDF. The default value is NONE. Possible values are {0}. Use together with -upwd and -opwd in case of PASSWORD mode, and -ec in case of CERTIFICATE
```